examples/research_projects/wav2vec2/run_pretrain.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/env python3
```

Comment:
Still testing this script with DeepSpeed to reproduce official results. This will probably take ~1 week or so. 
However, it's verified that loss decreases and that loss is identical to fairseq